frontend/src/components/Typography/Heading4/index.js

```diff
@@ -2,7 +2,7 @@ import { PropTypes } from "prop-types";
 import styles from "./index.module.css";
 
 const Heading4 = ({ children }) => {
-  return <h3 className={styles.heading3}>{children}</h3>;
+  return <h3 className={styles.heading4}>{children}</h3>;
```

Comment:
This is kinda of odd. The component is called `heading4` but it uses an `h3`. It would actually be nice to look into refactoring the `Heading` components into one.

Reply:
Ops my bad, it had the mistake of all says 3 when it's 4 and I forgot to fix that as well